src/views/answerView.js

```diff
@@ -4,10 +4,38 @@
  * Create an Answer element
  * @returns {Element}
  */
-export const createAnswerElement = (key, answerText) => {
+
+let score = 0
+export const createAnswerElement = (key, answerText , currentQuestion) => {
   const element = document.createElement('li');
   element.innerHTML = String.raw`
     ${key}: ${answerText};
   `;
+
+  element.classList.add('answer-item')
+  element.id = key
+
+  element.addEventListener('click' , myFun)
+
+  function myFun() {
+    const parent = element.parentNode
+    Array.from(parent.children).forEach(child => (child.classList = 'answer-item'))
+    
+    element.classList.add('selected')
+    currentQuestion.selected = key
+    
+    currentQuestion.correct === currentQuestion.selected && (score += 1)
```

Comment:
Although just one line I think this is quite difficult to read. A normal if statement would suffice here in my opinion

Reply:
done